diff -Naur -x '*~' tiki-6.0/installer/tiki-installer.php tiki-6.0-use-external-smarty/installer/tiki-installer.php
--- tiki-6.0/installer/tiki-installer.php	2010-11-03 11:27:58.000000000 +0000
+++ tiki-6.0-use-external-smarty/installer/tiki-installer.php	2010-12-10 15:45:07.000000000 +0000
@@ -17,7 +17,7 @@
 require_once( 'tiki-filter-base.php' );
 
 // Define and load Smarty components
-require_once ( 'lib/smarty/libs/Smarty.class.php');
+require_once ( 'smarty/Smarty.class.php');
 require_once ('installer/installlib.php');
 
 $dbTiki = false;
diff -Naur -x '*~' tiki-6.0/lib/init/smarty.php tiki-6.0-use-external-smarty/lib/init/smarty.php
--- tiki-6.0/lib/init/smarty.php	2010-09-27 18:44:28.000000000 +0000
+++ tiki-6.0-use-external-smarty/lib/init/smarty.php	2010-12-10 15:45:29.000000000 +0000
@@ -12,7 +12,7 @@
 }
 
 require_once 'lib/setup/third_party.php';
-require_once (defined('SMARTY_DIR') ? SMARTY_DIR : 'lib/smarty/libs/') . 'Smarty.class.php';
+require_once (defined('SMARTY_DIR') ? SMARTY_DIR : 'smarty/') . 'Smarty.class.php';
 
 class Smarty_Tikiwiki extends Smarty
 {
